css/quotes.css

```diff
@@ -1,4 +1,193 @@
-/* Quote page specific styles */
+body {
+    margin: 0;
+}
 
+.form {
+    margin-left: 5%;
+    margin-right: 5%;
+    max-width: 90%;
+    font-family: 'Roboto', sans-serif;
+}
 
-/* We should use a mobile first approach having out breack point at 768px */
\ No newline at end of file
+.form__title {
+    font-size: 22px;
+    font-weight: bold;
+    text-align: justify;
+    margin-top: 50px;
+    margin-bottom: 10px;
+    color: rgba(0, 0, 0, 0.7)
+}
+
+.form__subtitles {
+    font-weight: bold;
+    text-align: justify;
+    margin-bottom: 18px;
+    color: rgba(0, 0, 0, 0.7)
+}
+
+.hr__quote {
+    border: 0;
+    margin-left: 0%;
+    margin-right: 50%;
+    width: 90px;
+    height: 1px;
+    background-color: #0086c3;
+}
+
+.hr__details {
+    margin-bottom: 36px;
+}
+
+.form__paragraph {
+    font-size: 12px;
+    font-weight: lighter;
+    margin-left: 5%;
+    margin-right: 5%;
+    text-align: justify;
+    margin-bottom: 40px;
+}
+
+
+/*
+*In this code, i assign properties like border and color to the focused element.
+*/
+
+.form__input:focus {
```

Comment:
Please delete the space between Doc information and class name 😄 
Apply for all lines.